src/main/assets/scss/main.scss

```diff
@@ -155,6 +157,66 @@ $min-width: 52em;
   margin-right: 10px;
 }
 
+@media (min-width: 770px) {
+  .hide {
+    display: block;
+  }
+}
+
+.moj-button-menu {
+  display: none;
+}
+
+.alphabetical-box {
+  .filter-back-to-top-wrapper {
+    left: 100%;
+    position: relative;
+    width: 14%;
+  }
+
+  .layout-width-three-fifths {
+    width: 100%;
+  }
+}
+
+@media (max-width: 770px) {
+  .hide {
+    display: none !important; // sass-lint:disable-line no-important
+  }
+
+  .moj-button-menu {
+    display: block;
+  }
+
+  .moj-filter-layout {
+    &__filter {
+      width: 100%;
+    }
+  }
+
+  .filters {
+    &__innerButton {
+      display: block;
+      margin-left: 0;
+      margin-top: 0;
+      position: relative;
+    }
+  }
+
+  .alphabetical-box {
+    .layout-width-two-fifths  {
```

Comment:
I believe there is a neater way to achieve this using `layout-width-two-fifths-from-desktop` and `layout-width-three-fifths-from-desktop`
https://design-system.service.gov.uk/styles/layout/tablet-desktop/index.html